### PR TITLE
fix #133157 work.ink

### LINKED ISSUE #133157: work.ink
### Issue URL (Anti Adblock Script)
 [https://work.ink/21T/C401oid4IRc](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwork.ink%2F21T%2FC401oid4IRc)
### Comment
Username: @wernser412
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.37
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/133157

### Add your comment and screenshots

- https://github.com/AdguardTeam/AdguardFilters/issues/126208

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -1401,16 +1401,22 @@ strtapeadblocker.art#@#.skyscraper.ad
 hotdebrid.com#%#//scriptlet('prevent-element-src-loading', 'script', 'ads.rubiconproject.com/ad/')
 !+ PLATFORM(ios, ext_android_cb)
 @@||ads.rubiconproject.com/ad/*.js$domain=hotdebrid.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/123316
+! https://github.com/AdguardTeam/AdguardFilters/issues/133157
+work.ink#?#.content .col-md .row:has(div[id^="google_ads_"])
```
Comment (Alex-302) [inline BaseFilter/sections/antiadblock.txt:1405]: But this is an ad(ad leftover)

Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:1405]: Yeap. I added the rule to block 2 leftovers on the website.

**Comment by piquark6046** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/133245#issuecomment-1291932616
Rechecked.

**Review by zloyden — COMMENTED** [review]
@AdamWr this rule also works on my end.
```
work.ink#%#//scriptlet('prevent-setTimeout', 'sas_26322')
```

**Comment by AdamWr** [comment]
> @AdamWr this rule also works on my end.
> 
> ```
> work.ink#%#//scriptlet('prevent-setTimeout', 'sas_26322')
> ```

Yes, it works fine on my end too :)

**Review by zloyden — COMMENTED** [review]
@piquark6046 any updates on this?
I suggest you add the scriptlet https://github.com/AdguardTeam/AdguardFilters/pull/133245#pullrequestreview-1156423507 and exceptions under the directive for content blockers.
Something like that
```
work.ink#%#//scriptlet('prevent-setTimeout', 'sas_26322')
!#if (adguard_ext_android_cb)
@@||cmp.quantcast.com/choice/*/choice.js$domain=work.ink
@@||cmp.quantcast.com/tcfv2/*/cmp2.js$domain=work.ink
@@||cdn.thisiswaldo.com/static/js/11929.js$domain=work.ink
@@||cdn.taboola.com/libtrc/workink/loader.js$domain=work.ink
@@||securepubads.g.doubleclick.net/tag/js/gpt.js$domain=work.ink
@@||adservice.google.com/adsid/$domain=work.ink
@@||adservice.google.co.*/adsid/$domain=work.ink
work.ink#@##AdHeader
work.ink#@##AD_Top
work.ink#@##homead
work.ink#@##ad-lead
!#endif
```

**Comment by piquark6046** [comment]
Are non-scriptlet rules necessary for AdGuard for iOS (no advanced protection)?
@zloyden

**Comment by zloyden** [comment]
Actually, scriptlets are supported by AdGuard for iOS, but if iOS >= 15. as I recall.
You can extend the directive as
```
!#if (adguard_ext_android_cb || adguard_app_ios || adguard_ext_safari)
```

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [inline BaseFilter/sections/antiadblock.txt:1405]: Where do you need the rule?

Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt]: The rule had been tested when I wrote the exception rules.
However, the rule has been become useless due to the scriptlet rule.
I added a rule hiding leftovers on the website into `css_extended.txt`.

File: BaseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -6,6 +6,8 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/133157
+work.ink#?#.content .wk-container:has(.container-content:not(:has(> *)))
```
Comment (zloyden) [inline BaseFilter/sections/css_extended.txt:10]: It does not block leftovers on my end.
<details><summary>Screenshot:</summary>

image

</details><br/>

Try this `work.ink#?#.wk-container:has(> div.container-content > div[id^="waldo-tag-"]:first-child)`

Comment (piquark6046) [reply BaseFilter/sections/css_extended.txt:10]: So strange. I see the website as the following screenshot:

Screensh1
Screenshot from 2022-10-31 09-13-11

Comment (zloyden) [reply BaseFilter/sections/css_extended.txt:10]: Yep, it is. Did the rule above help on your end?

Comment (piquark6046) [reply BaseFilter/sections/css_extended.txt:10]: But, your extended css rule works on my end too.
I will use yours.

**Review by zloyden — APPROVED** [review]

**Pull request merged**